postgres/sql/users.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE IF NOT EXISTS public."Users"
+(
+    first_name text COLLATE pg_catalog."default",
+    last_name text COLLATE pg_catalog."default",
+    role text COLLATE pg_catalog."default",
```

Comment:
Please remove the role field. We won't need to store roles because all users that are stored in the DB are admins.